src/domain/follows_differ.rs

```diff
@@ -357,13 +352,19 @@ fn log_line(
 
     // Investigate states in which there are no followees but there are unfollowed followees
     if followed_counter == 0 && unfollowed_counter > 0 && unchanged == 0 {
+        metrics::sudden_follow_drops().increment(1);
+
+        if has_nos_agent(event) {
+            metrics::nos_sudden_follow_drops().increment(1);
+        }
+
         return Some(format!(
             "ALL UNFOLLOWED: Npub {}: date {}, {} unfollowed, {} unchanged, {}",
             follower.to_bech32().unwrap_or_default(),
             timestamp_diff,
             unfollowed_counter,
             unchanged,
-            event.as_json()
+            event.as_json(),
         ));
```

Comment:
This is what we log when this happens, we can filter by the `nos` client, but we log for any client tag when this situation arises.